share/style.css

```diff
@@ -304,7 +309,63 @@ html, body {
 	vertical-align: top;
 }
 
-/* link, list, paragraph, table */
+.osu-md__link--footnote-ref {
+	font-size: 11px;
+}
+
+.osu-md__link--footnote-ref:after {
+	content: ']';
+}
+
+.osu-md__link--footnote-ref:before {
+	content: '[';
+}
+
+.osu-md ul,
+.osu-md ol {
+	padding-left: 20px;
+}
+
+.osu-md ol {
+	counter-reset: c var(--list-start);
+	padding-left: 0;
+	list-style: none;
+}
+
+.osu-md li {
+	margin: 5px 0;
+}
+
+.osu-md ol > li {
+	display: flex;
+	counter-increment: c;
+}
+
+.osu-md ol > li:before {
+	flex: none;
+	padding-right: 5px;
+	min-width: 30px;
+	content: counter(c) '. ';
+}
+
+.osu-md ol > li > div {
+	flex: 1;
+	min-width: 0;
+}
+
+.osu-md p {
+	margin-bottom: var(--paragraph-space);
+	line-height: 1.5;
+}
```

Comment:
this kind of thing is probably going to need to stick with the classnames from the original css to be accurate. for example, the `figure-container` matches this selector, but on the website's version it wouldn't.